Servers/Tomcat v9.0 Server at localhost-config/catalina.policy

```diff
@@ -0,0 +1,260 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Are you sure that this file is required to be on the repository? 

FYI To the repository, you should add project files, that are responsible for project realization. For example: if you are using some library, then you don't need to add it to the repository, as the library will be the same for all, if you will use the same library version and download from the same repository.

So the main idea to save on repository files and configurations. For git you can find and use `.gitignore` file to skip adding not necessary files to the project with examples ;)